lesson21/main.js

```diff
@@ -119,23 +124,92 @@ const formingTableHeaderNameWithKey = (key) => {
 const createTableData = (usersData) => {
   const fragment = document.createDocumentFragment();
   usersData.forEach((userData) => {
-    const trOfTdata = document.createElement("tr");
+    const trOfTdata = createElementWithClassName("tr", "users-table__tr-td");
     /**
      *Extract value from each of usersData
      * @param {Object} userData the object of each users data
      * @param {String,Number} val value of each users data
      */
 
-    Object.values(userData).forEach((val) => {
+    Object.keys(userData).forEach((key) => {
+      if (key === "id") return;
```

Comment:
Nice write up!

Reply:
Thank you! It thanks to your teaching before :)